permission-portal-frontend/client/src/screens/Login.js

```diff
@@ -51,20 +52,21 @@ const SignInFormBase = observer(
       <Fragment>
         <div className="doctorContainer">
           <div className="doctor1">
-            <img src={doctor1} />
+            <img src={doctor1} alt="" />
           </div>
           <div className="doctor2">
-            <img src={doctor2} />
+            <img src={doctor2} alt="" />
           </div>
         </div>
       </Fragment>
     )
 
     loginForm = () => (
       <Fragment>
+        <PageTitle title="Welcome" />
         <div className="topNav">
-          <img src={ucsf_health} id="ucsfLogo" />
-          <img src={powered_by_cw} id="poweredByCWLogo" />
+          <img src={ucsf_health} id="orgLogo" alt={this.props.store.organization.name || 'UCSF Health'} />
```

Comment:
This will always return "UCSF Health" because `store.organization.name` is always blank on this page.

Reply:
ah that seems like a bug